aws/resource_aws_opsworks_application.go

```diff
@@ -312,24 +310,9 @@ func resourceAwsOpsworksApplicationCreate(d *schema.ResourceData, meta interface
 		Attributes:       resourceAwsOpsworksApplicationAttributes(d),
 	}
 
-	var resp *opsworks.CreateAppOutput
-	err = resource.Retry(2*time.Minute, func() *resource.RetryError {
-		var cerr error
-		resp, cerr = client.CreateApp(req)
-		if cerr != nil {
-			log.Printf("[INFO] client error")
-			if opserr, ok := cerr.(awserr.Error); ok {
-				// XXX: handle errors
-				log.Printf("[ERROR] OpsWorks error: %s message: %s", opserr.Code(), opserr.Message())
-				return resource.RetryableError(cerr)
```

Comment:
It appears this `resource.RetryableError()` usage may have been legitimate because unlike the `UpdateApp` usage below, its retrying on AWS Go SDK errors. That said, it is probably okay to remove this and see if anything comes up where we discover the _appropriate_ errors (if any) to actually retry on. The acceptance testing is not finding it currently.